addons/arcadian/base/config_animationsources.hpp

```diff
@@ -0,0 +1,35 @@
+class AnimationSources: AnimationSources {
+    class HitGlass1;
+    class HitGlass7: HitGlass1 {
+        hitpoint = "HitGlass7";
+    };
+    class HitGlass8: HitGlass1 {
+        hitpoint = "HitGlass8";
+    };
+    class reardoor_2_source {
+        displayName = "Rear Door Window";
```

Comment:
Less important than `UserActions`, but could also stringtable those (especially if strings already exist in vanilla).